Makefile

```diff
@@ -260,7 +260,15 @@ release-manager-snapshot: release
 .PHONY: release-manager-release
 release-manager-release: release
 
+JAVA_ATTACHER_VERSION:=1.26.0
+JAVA_ATTACHER_JAR:=apm-agent-attach-cli-$(JAVA_ATTACHER_VERSION).jar
+JAVA_ATTACHER_URL:=https://search.maven.org/remotecontent?filepath=co/elastic/apm/apm-agent-attach-cli/$(JAVA_ATTACHER_VERSION)/$(JAVA_ATTACHER_JAR)
+
+build/$(JAVA_ATTACHER_JAR):
+	curl -sSL $(JAVA_ATTACHER_URL) > $@
```

Comment:
Can we verify the content somehow?

@felixbarny are there any signatures available for verification?

Otherwise, we could just hardcode a sha256 hash alongside JAVA_ATTACHER_VERSION.

Reply:
https://repo1.maven.org/maven2/co/elastic/apm/apm-agent-attach-cli/1.26.0/ looks like we have a few different options (md5, sha1, .asc). I'll add that

Reply:
It's best to validate against the `.asc` (pgp) key as it doesn't need to change if you update versions. If that's not feasible, prefer sha1 over md5. Also, you'll have to store the key in a file or in a hard-coded string. Otherwise, a man-in-the-middle attack that injects a malicious attacher could just as well spoof the corresponding hash.

Reply:
The agent's public key is here: https://github.com/elastic/apm-agent-java/blob/master/apm-agent-attach-cli/src/main/resources/public_key.asc